torch/signal/windows/windows.py

```diff
@@ -759,3 +732,67 @@ def general_hamming(M,
                           layout=layout,
                           device=device,
                           requires_grad=requires_grad)
+
+
+@_add_docstr(
+    r"""
+Computes the minimum 4-term Blackman-Harris window according to Nuttall.
+
+.. math::
+    w_n = 1 - 0.36358 \cos{(z_n)} + 0.48917 \cos{(2z_n)} - 0.13659 \cos{(3z_n)} + 0.01064 \cos{(4z_n)}
```

Comment:
@lezcano this formula computes each window timestep, no need to add `sum`.

Reply:
You are completely right. My bad!